src/Store/SessionStore.php

```diff
@@ -137,6 +137,12 @@ public function save(HTTPRequest $request)
         return $this;
     }
 
+    /**
+     * Clear any stored values for the given request
+     *
+     * @param HTTPRequest $request
+     * @return void
```

Comment:
You can use `{@inheritDoc}` from the interface instead of specifying these params (keep the message though)